### PR TITLE
Remove useless code

### PR DESCRIPTION
**Pull request by bzheng888**
### What changes are proposed in this pull request?
Remove useless continue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Review by maobaolong — APPROVED** [review]
@bzheng888 Thanks for simplify the code. It LGTM.

**Comment by maobaolong** [comment]
Hi @beinan would you like to take another look?

File: core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java [hunk 1]
```diff
@@ -834,7 +834,6 @@ public synchronized void freeSpace(long sessionId, long minContiguousBytes,
           spaceFreed += blockMeta.getBlockSize();
         } catch (BlockDoesNotExistException e) {
           LOG.warn("Failed to evict blockId {}, it could be already deleted", blockToDelete);
-          continue;
```
Comment (LuQQiu) [inline core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java:837]: why this one is not needed? continue to evict other blocks which may exist looks correct?

**Review by LuQQiu — CHANGES_REQUESTED** [review]
The `continue` keyword looks useful and correct

File: core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java (hunk 1 above)
Comment (LuQQiu) [reply core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java:837]: This continue is for the outer while(true) loop which should stop when eviction finished or no any block that is available to evict

**Comment by bzheng888** [comment]
@LuQQiu The code is updated,  it seems no  problem now.
image

File: core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java (hunk 1 above)
Comment (beinan) [reply core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java:837]: +1, I'm also thinking this continue is necessary to evict other blocks

**Review by beinan — CHANGES_REQUESTED** [review]

**Comment by bzheng888** [comment]
@LuQQiu @beinan This pr was brought up a long time ago, it looks like someone fixed it later, it's ok now

**Comment by beinan** [comment]
> @LuQQiu @beinan This pr was brought up a long time ago, it looks like someone fixed it later, it's ok now

Got it, thank you for looking into this! so shall we close this pr?

**Comment by bzheng888** [comment]
> > @LuQQiu @beinan This pr was brought up a long time ago, it looks like someone fixed it later, it's ok now
> 
> Got it, thank you for looking into this! so shall we close this pr?

Yes, I'm going to close it!

**Pull request closed without merging**